dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.35/V2_35_13__Add_description_deg_and_indg.sql

```diff
@@ -0,0 +1,2 @@
+alter table dataelementgroup add column if not exists "description" text;
```

Comment:
Remember to register the Flyway migration here:

https://github.com/dhis2/wow-backend/blob/master/coordination/flyway_versioning.md

Next available version is currently V2_35_15.

The integration tests now fail with:

Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flyway' defined in class path resource [org/hisp/dhis/db/migration/config/FlywayConfig.class]: Invocation of init method failed; nested exception is org.flywaydb.core.api.FlywayException: Found more than one migration with version 2.35.13

Reply:
Yes, now fixed.